package.json

```diff
@@ -23,6 +23,7 @@
     "github": "^5.2.0",
     "js-yaml": "^3.6.1",
     "json-parameterization": "^0.2.0",
+    "jsonwebtoken": "https://registry.npmjs.org/jsonwebtoken/-/jsonwebtoken-5.7.0.tgz",
```

Comment:
Oh, I don't know - this line happened after `npm install --save jsonwebtoken` . You mean, we should just replace this with a version number, as in other dependencies?